### PR TITLE
falcoctl v0.5.1 (new formula)

### LINKED ISSUE #252: Add falcoctl to brew
**What would you like to be added**:

A homebrew formula to be able to install falcoctl with `brew`

**Why is this needed**:

To make osx folks happy installing falcoctl directly from brew.

### PR DESCRIPTION
**Pull request by cpanato**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Fixes: https://github.com/falcosecurity/falcoctl/issues/252

### CODE REVIEW

File: Formula/f/falcoctl.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      -X #{pkg}.buildDate=#{time.iso8601}
+      -X #{pkg}.gitCommit=#{Utils.git_head}
+      -X #{pkg}.semVersion=#{version}
+    ]
+
+    system "go", "build", *std_go_args(ldflags: ldflags), "."
+  end
+
+  test do
+    assert_match version.to_s, shell_output(bin/"falcoctl version 2>&1")
```
Comment (chenrui333) [inline Formula/f/falcoctl.rb:25]: need a better test than version check

File: Formula/f/falcoctl.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class Falcoctl < Formula
+  desc "Container Signing"
```
Comment (chenrui333) [inline Formula/f/falcoctl.rb:2]: looks like copy paste issue 😄

File: Formula/f/falcoctl.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  homepage "https://github.com/falcosecurity/falcoctl"
+  url "https://github.com/falcosecurity/falcoctl.git",
+      tag:      "v0.5.1",
+      revision: "cc49e6c6e3d2353e937b628ba33ce3e6a8c55b95"
+  license "Apache-2.0"
+  head "https://github.com/sigstore/cosign.git", branch: "main"
```
Comment (chenrui333) [inline Formula/f/falcoctl.rb:8]: same as this one.

**Review by chenrui333 — APPROVED** [review]

**Review by ZhongRuoyu — APPROVED** [review]
Thanks, @cpanato and @chenrui333.

**Comment by cpanato** [comment]
@chenrui333 thanks so much for the updates

**Pull request merged**